README.md

```diff
@@ -815,25 +815,28 @@ Happy Coding! -->
   <td align="center"> 261.</td>
   <td align="center"><a href="https://github.com/pranjay-poddar/Dev-Geeks/tree/main/Js-Games/Mastermind">Mastermind</a></td>
   <td align="center"> 268.</td>
-
   <td align="center"> <a href="https://github.com/pranjay-poddar/Dev-Geeks/tree/main/Python Projects/Forex Exchanger">Web-Image Filter</a></td>
   <td align="center"> 272.</td>
   <td align="center"> <a href="https://github.com/pranjay-poddar/Dev-Geeks/tree/main/Js-Games/Cubula">Cubula</a></td>
-
+ </tr>
+  
+ <tr align="Center">
```

Comment:
'c' in the "center" should be in lowercase

Reply:
image.png

It's correct

Reply:
Done👍🏼